system/include/webgpu/webgpu.h

```diff
@@ -465,6 +515,8 @@ typedef enum WGPUTextureUsage {
     WGPUTextureUsage_Sampled = 0x00000004,
     WGPUTextureUsage_Storage = 0x00000008,
     WGPUTextureUsage_OutputAttachment = 0x00000010,
+    WGPUTextureUsage_RenderAttachment = 0x00000010,
+    WGPUTextureUsage_Present = 0x00000020,
```

Comment:
This is from Dawn's old native presentation code which hasn't been removed yet.
```suggestion
```

Reply:
Removed